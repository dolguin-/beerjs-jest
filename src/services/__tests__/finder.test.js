const finder = require('../finder');

describe('Finder', () => {
  it('should match the snapshot', () => (
    finder('rosario')
    .then((response) => {
      expect(response).toMatchSnapshot();
    })
  ));
});
