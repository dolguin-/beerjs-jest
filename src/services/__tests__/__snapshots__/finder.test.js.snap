// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finder should match the snapshot 1`] = `
Object {
  "data": Object {
    "hello": "world",
  },
}
`;
